frontend/views/components/menu/MenuItem.vue

```diff
@@ -16,7 +16,7 @@ export default ({
   inheritAttrs: false,
   methods: {
     handleSelect () {
-      this.$listeners.click && this.$listeners.click()
+      // this.$listeners.click && this.$listeners.click()
```

Comment:
Why is this being commented out?

Which issue is this addressing?

Reply:
It's for the second Issue I pointed [here](https://github.com/okTurtles/group-income/issues/1574#issuecomment-1541196105). (_The other is that makeEmoticon event is emitted twice._)
I found that `click` event is handled twice in all the MenuItems so I commented this out. Should I remove this?